src/main/kotlin/day03/Hero.kt

```
@@ -0,0 +1,30 @@
+    val slime = Slime(100, "A")
+    println(slime.suffix)
+    println(slime.hp)
+}
+
+class Hero(
+    var name: String,
+    var hp: Int,
+) {
+    fun attack() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)